__tests__/intergration.test.js

```diff
@@ -70,12 +70,43 @@ describe("GET /api", () => {
     test("each API property should be correct type", () => {
         return request(app).get("/api").expect(200).then(result => {
             const returnObj = result.body.API;
-            console.log(returnObj);
             for (const key in returnObj) {
                 expect(typeof returnObj[key].description).toBe("string")
                 expect(Array.isArray(returnObj[key].queries)).toBe(true)
                 expect(typeof returnObj[key].exampleResponse).toBe("object")
             }
         })
     })
+});
+describe.only("Get article API", () => {
+    test("Should return status 200 if article exists", () => {
+        return request(app).get("/api/articles/2").expect(200);
+    })
+    test("If article ID exists should return the article object found", () => {
+        return request(app).get("/api/articles/2").expect(200).then(result => {
+            expect(result.body.article).toMatchObject([
+                {
+                    article_id: 2,
+                    title: 'Sony Vaio; or, The Laptop',
```

Comment:
we should make this dynamic by expecting a string etc rather than the actual body etc the only one we can expect is that article_id is 2